### PR TITLE
New SMES sprites by Irra

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request

New SMES sprites by Irra or @martinlyra from https://github.com/Haven-13/Haven-Urist/pull/67
SMES's have a EMP interaction now.

## Why It's Good For The Game

image
image

## Changelog
:cl: Irra 
balance: SMES's now have a EMP interaction 
imageadd: Smes sprites
/:cl:

### CODE REVIEW

**Comment by MarkSuckerberg** [comment]
@triplezeta

**Comment by martinlyra** [comment]
Just to set some context for the `failing` icon-state. Originally in the Baycode when I made these sprites, it was a state where the SMES was about to explode; destroying itself and its surroundings. It would be the warning sign while shaking prior for some time before exploding.

I'd recommend making a new state for the broken state as the current failing state expresses immediate danger, when it really is not going to.

**Comment by rye-rice** [comment]
> Just to set some context for the `failing` icon-state. Originally in the Baycode when I made these sprites, it was a state where the SMES was about to explode; destroying itself and its surroundings. It would be the warning sign while shaking prior for some time before exploding.
> 
> I'd recommend making a new state for the broken state as the current failing state expresses immediate danger, when it really is not going to.

Removed the failing overlay, I thought it was used when the smes'es were shut down by power check event.

**Comment by Fikou** [comment]
> balance: SMES's now have a interaction

thank you great changelog

**Comment by rye-rice** [comment]
i mean, yeah

**Review by triplezeta — APPROVED** [review]

File: code/modules/power/smes.dm [hunk 1]
```diff
@@ -208,25 +213,56 @@
...
+		. += "charge[clevel]"
+		SSvis_overlays.add_vis_overlay(src, icon, "charge[clevel]", layer, plane, dir)
+		SSvis_overlays.add_vis_overlay(src, icon, "charge[clevel]", layer, EMISSIVE_PLANE, dir)
+	if(is_emped)
+		. += "emp"
+		SSvis_overlays.add_vis_overlay(src, icon, "emp", layer, plane, dir)
+		SSvis_overlays.add_vis_overlay(src, icon, "emp", layer, EMISSIVE_PLANE, dir)
+	else
+		if(inputting)
+			if(clevel == 5)
```
Comment (Fikou) [inline code/modules/power/smes.dm]: use the clevel defines

**Comment by PrefabQuasar** [comment]
i think this PR caused the overlays for shuttle engine SMES units to stop working

**Pull request merged**